fix(accessibilité): amélioration du header

m-maillot opened the pull request:
closes #6289

maxgfr commented on packages/code-du-travail-frontend/src/modules/layout/header/HeaderBrand.tsx:
```diff
@@ -15,11 +15,11 @@ export const HeaderBrand = () => {
               className={`${fr.cx("fr-responsive-img")} ${Logo}`}
               src="/static/assets/img/logo.svg"
-              alt="Accueil - Code du travail numérique, Ministère du Travail"
+              alt=""
```
on peut le supprimer ou alors on le garde

m-maillot replied:
Il faut le mettre à vide. Si je le retire, il ne sera pas présent et j'ai peur que ce ne soit pas bon...

Viczei commented:
> This PR attempts to improve header accessibility by moving descriptive text from the image alt attribute to the link title attribute. While the intent is good, this approach may reduce accessibility for some users since title attributes are not consistently announced by all screen readers and assistive technologies.

L'ia n'a pas l'air de trouver ca accessible :)

m-maillot committed:
fix(accessibilité): amélioration du header

# Conflicts:
#	packages/code-du-travail-frontend/src/modules/layout/header/HeaderBrand.tsx
#	packages/code-du-travail-frontend/src/modules/layout/header/__tests__/__snapshots__/HeaderDsfr.test.tsx.snap
